scripts/macosx/build_shout.sh

```diff
@@ -27,11 +27,6 @@ do
   tar -zxf $DEPENDENCIES/$ARCHIVE -C $VERSION-$ARCH --strip-components 1
   cd $VERSION-$ARCH
 
-  # Apply patch fixing: 
-  # https://trac.xiph.org/ticket/2244
-  # https://bugs.launchpad.net/mixxx/+bug/1544739
-  patch -p1 < $PROGDIR/fix_libshout_ticket2244.patch
-
```

Comment:
Did you remove this because we now link libshout statically with the patch applied? Why do we even need this at all? I thought we have libshout in our lib directory.

Reply:
The patch was for a bug that has since been fixed upstream.

Reply:
Yes. Since https://github.com/mixxxdj/mixxx/pull/2040 we have libshout in the lib folder. Can't we just use that on macOS/Windows, instead of adding it to build env?

Reply:
We should only be relying on the `lib` directory in the source repository for exceptional situations. We should remove libshout from there when Linux distros ship a fixed version. I do not see any good reason to remove libshout from the macOS or Windows.

Reply:
ok